tile.py

```diff
@@ -0,0 +1,71 @@
+import json
+import pygame
+
+
+class Tile:
```

Comment:
Make both minefield and tile inherit a 'Drawable' class. I'll let you decide how it needs to be written